### PR TITLE
Nisha | Test | Version 2 Varaint wise export report

### PR DESCRIPTION
**Pull request by nisha-walkover**
https://app.clickup.com/t/86cvh7cnb

### CODE REVIEW

File: apps/web-giddh/src/app/new-inventory/component/inventory.store.ts [hunk 1]
```diff
@@ -67,4 +67,42 @@ export class InventoryComponentStore extends ComponentStore<any> {
...
+    readonly exportVariant = this.effect((data$: Observable<{ stockReportRequest: any, queryParams: any }>) => {
+        return data$.pipe(
+            switchMap((req) => {
+                this.patchState({ isLoading: true });
+                return this.inventoryService.getVariantWiseReportExport(req.queryParams, req.stockReportRequest).pipe(
+                    tapResponse(
+                        (res: BaseResponse<any, any>) => {
+                            if (res.status === "success") {
+                                if (typeof res?.body === "string") {
+                                    this.toaster.showSnackBar("success", res?.body);
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/new-inventory/component/inventory.store.ts]:
```suggestion
                                    this.toaster.showSnackBar("success", res.body);
```

File: apps/web-giddh/src/app/new-inventory/component/inventory.store.ts [hunk 2]
```diff
@@ -67,4 +67,42 @@ export class InventoryComponentStore extends ComponentStore<any> {
...
+                                    this.router.navigate(["/pages/downloads"]);
+                                    return this.patchState({
+                                        isLoading: false
+                                    });
+                                }
+                            } else {
+                                this.toaster.showSnackBar("error", res?.message);
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/new-inventory/component/inventory.store.ts]:
```suggestion
                                res?.message && this.toaster.showSnackBar("error", res.message);
```

File: apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.ts [hunk 3]
```diff
@@ -365,6 +366,12 @@ export class ReportFiltersComponent implements OnInit, OnChanges, OnDestroy {
             else if (column === 'closing_amount') {
                 this.stockReportRequestExport.showClosingStockValue = true;
             }
+            /* for value filter selected in variant */
+            if(InventoryReportType.variant) {
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.ts]: format code

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

**Review by dvCodeWorld — APPROVED** [review]

File: apps/web-giddh/src/app/new-inventory/component/inventory.store.ts (hunk 1 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/new-inventory/component/inventory.store.ts]: done

File: apps/web-giddh/src/app/new-inventory/component/inventory.store.ts (hunk 2 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/new-inventory/component/inventory.store.ts]: done

File: apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.ts (hunk 3 above)
Comment (nisha-walkover) [reply apps/web-giddh/src/app/new-inventory/component/report-filters/report-filters.component.ts]: done

**Pull request merged**